app/controllers/api/account.php

```diff
@@ -1845,6 +1845,9 @@
             'team' => '',
         ];
 
+        // Hide secret for clients
+        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $tokenSecret : '');
```

Comment:
This will remove the secret from other destinations like functions and webhooks, too won't it?

Reply:
Yep, will fix